delayed_job/monitor_delayed_jobs.rb

```diff
@@ -10,14 +10,13 @@ class MonitorDelayedJobs < Scout::Plugin
     default: production
   queue_name:
     name: Queue Name
```

Comment:
What about these then? You've done it for custom loader. On phone so not looking carefully.